app/src/main/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerViewModel.kt

```diff
@@ -24,6 +24,7 @@ class QuestionPlayerViewModel @Inject constructor() : ObservableViewModel() {
   val progressPercentage = ObservableField(0)
   val isAtEndOfSession = ObservableBoolean(false)
   private val canSubmitAnswer = ObservableField(false)
+  var isAnswerCorrect = false
```

Comment:
Suggest using `val` instead of `var`. or use an `ObservableBoolean `

Reply:
isAnswerCorrect is being reassigned in QuestionPlayerFragmentPresenter based on if the answer by the user is correct or incorrect.

Reply:
If its getting reassigned then you can use 
`val isAnswerCorrect = ObservableField<Boolean>(false)`

and in this you can set and get values like
`isAnswerCorrect.get()`

`isAnswerCorrect.set(true)`

Reply:
@MaskedCarrot PTAL

Reply:
done